vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -1,52 +1,76 @@
-- The [Architecture](link) documentation details the components of the PDA plugin architecture.
-- The [Customization and Extension](link) documentation describes how to customize and build upon the PDA plugin.
-- The [Technical Insights](link) tutorial provides a detailed technical overview of how to add new functionality to a PDA implementation.
- 
-# Installation
- 
-# Configuration
- 
-# Architecture
-This document provides a comprehensive description of the building blocks used to create the PDA plugin. These include the following components:
- 
-- Micro Frontend
+---
+sidebarDepth: 2
+---
+
+# Entando Process Driven Applications Plugin
+
+## Introduction
+
+The Entando Process Driven Applications (PDA) plugin is engineered to provide a rich and full-featured user experience while facilitating the management and completion of business processes and automation.  The solution template includes: 
+
+- A general purpose UX layer that is created from micro frontends and can be implemented for any business process or task engine. The UX layer can serve data via the included Entando integration adapter or by implementing a set of interfaces on the server side.
+- A Spring Boot microservice backend providing a pluggable interface for the injection of underlying processes or automation toolkits. The interfaces and steps for creating a new PDA backend implementation are described in further detail [here](link).
+
+This tutorial will guide you through installing the PDA plugin using the Entando Component Repository (ECR) and an Entando bundle. The goal of this exercise is to demonstrate the scope of process automation functionality enabled by integrating the following components: 
+
+- custom Micro Frontends
 - Process Automation Manager
-- Business Process Modeling Notation
-- Decision Model and Notation
 - Backend for Frontend
+
+Key elements of the template are reviewed in the [Application Details section](#application-details) below.
+
+## Installation
+
+There are numerous assets installed as part of the Entando PDA plugin. Entando Bundles can include more or less components, depending on objectives. It is recommended that organizations develop guidelines for bundle sizing that fit the goals of their applications and teams.
+
+### Prerequisites
+
+- An Entando Application on any Kubernetes provider. Follow one of the [tutorials](../#operations) appropriate to your environment to install the Entando platform.
+- The ent command line tool, installed and connected to your Kubernetes instance.
+
+### Installation Steps
+
+1. Apply the definitions for the bundle that comprises the PDA plugin. You'll need to adjust the `-n entando` option in each command to match your namespace or project.
+```
+ent bundler from-git -r https://github.com/entando-samples/entando-process-driven-plugin-bundle.git -d | ent kubectl apply -n entando -f -
+```
+
+2. Log into your App Builder instance.
```

Comment:
if the instructions are more complex elsewhere, maybe we should include the periods. I've been bothered by this.

Reply:
...should not? or should?

Reply:
Probably should discuss in style meeting